.github/workflows/build-test.yml

```diff
@@ -317,20 +317,15 @@ jobs:
       matrix:
         jupyter: [raw, local]
         python: [nonConda, conda]
-        pythonVersion: [3.9]
+        pythonVersion: ['3.9', '3.10']
```

Comment:
I believe these are going to fail? Or maybe Ian hasn't submitted his new test yet.

Reply:
I would expect it to fail. At least it did for me on my local machine. Didn't try on CI yet though.

Reply:
I have filed an issue and yes, it will fail for Python 3.10.
I'll skip the test if using Python 3.10

Reply:
Test validated :). 
![image](https://user-images.githubusercontent.com/812783/145650365-65d4cc0b-d0fd-47a7-a2df-d896fc638bcc.png)
![image](https://user-images.githubusercontent.com/812783/145650390-b49645be-0073-4b47-8d4b-e9b02e180482.png)